test/move_effect_recoil_if_miss.c

```diff
@@ -40,10 +40,10 @@ SINGLE_BATTLE_TEST("Jump Kick has 50% recoil on protect")
     }
 }
 
-SINGLE_BATTLE_TEST("Jump Kick has no recoil if no target")
+SINGLE_BATTLE_TEST("Jump Kick has no recoil if no target") // Previously issue #2596.
```

Comment:
We don't need this comment, I just wrote the issue number down so that somebody could look up why the test was failing (because otherwise you might think it was a Jump Kick bug).

I'm very pleased this test passes now! :heart: